app/components/summary_list_component.rb

```diff
@@ -65,4 +67,8 @@ def format_list_with_bullets(list)
     end
     markup + '</ul>'.html_safe
   end
+
+  def format_list_as_paragraphs(list)
+    list.map { |s| "<p class=\"govuk-body\">#{ERB::Util.html_escape(s)}</p>" }.join.html_safe
```

Comment:
I had to go back to the original commit https://github.com/DFE-Digital/apply-for-teacher-training/commit/85f5ceb48dcdaff5d22520ba7cce7953f1fb5406#diff-d5ad8a1eabad78f51e9f7abe6236778e8b083340a7c7a73c1ee308d62b3eeec5 to understand why we are doing this (+1 for the commit message). I think this is ok, but not for this PR generally it would be good to revisit this rule across the app, we should probably choose one or the other for rendering the list for consistency.

Reply:
Raised a card to track the subsequent investigation: https://trello.com/c/Pv5CcCDW/482-review-summarylistcomponent-array-values-markup